### DISCUSSION TITLE
Kick-off call for the Misq project

### DISCUSSION THREAD

**Discussion by chimp1984** [bisq-network/bisq#5555] [General]
I would like to make a public jitsi conference call on <strike>Thursday 10. 6. 2021 at CET 17:00</strike> (EDIT: postponed to Friday 18. 6. 2021 at CET 17:00) about the Misq project.

It will cover those topics:
- What is Misq?
- Why we started to work on Misq?
- Who is working on it?
- What is the current state?
- How will it interact with Bisq?
- What is still open from conceptual side?
- What is mostly needed?
- Open discussion

Here are the links to the original proposal and the project page:
https://github.com/bisq-network/proposals/issues/330
https://github.com/bisq-network/projects/issues/51

The call will be recorded and published on our Youtube channel. People can join without audio/video as well.

**Comment by chimp1984** [comment]
Btw. I get my vaccination tomorrow, so in case if I would have severe side-effects I might need to postpone the call.

**Comment by chimp1984** [comment]
I have to postpone the kick-off call as I have side effects of the vaccine. Nothing severe, in the expected range (a bit flu like). I will post another date once I am fit again.

**Comment by chimp1984** [comment]
New date for the kick-off call is Friday 18. 6. 2021 at CET 17:00

**Reply by stopfeds** [reply]
Will you provide a link or how does that work? Sorry, never used Jitsi.

**Reply by chimp1984** [reply]
It will be Youtube at the end and I will post the link before the call here and on keybase.

**Comment by chimp1984** [comment]
Here is he link: https://jitsi.wiz.biz/KindPerspectivesStareImpartially

EDIT: Sorry link has changed: https://jitsi.wiz.biz/WrongInfluencesOccupyHappily

**Comment by chimp1984** [comment]
Recording of the call is here: https://youtu.be/TNsvOM07owg?t=321

**Reply by m52go** [reply]
I've replaced that video with a new one since it had some extra fluff in the beginning.

See this link:
https://bitcointv.com/videos/watch/f8dd2b42-3cd8-4a4a-ba64-d8a865379176

Or YouTube, if you must:
https://www.youtube.com/watch?v=UOarJpbg4EY
